resources/public/api/conf/3.0/examples/broughtForwardLosses/amendBFLoss/amend_se_class4_response.json

```diff
@@ -0,0 +1,14 @@
+{
+  "businessId": "XBIS12345678910",
+  "typeOfLoss": "self-employment-class4",
+  "lossAmount": 3456.78,
+  "taxYearBroughtForwardFrom": "2022-23",
+  "lastModified": "2020-07-13T12:13:48.763Z",
+  "links": [
+    {
+      "href": "/individuals/losses/TC663795B/brought-forward-losses/1234568790ABCDE",
```

Comment:
Is this the old Hateoas link? Should V3 not contain Retrieve BFL, Amend BFL and Delete BFL